members/WIL000347.yaml

```diff
@@ -21,16 +21,26 @@ contact_form:
           selector: div.citystatezipcode input
           value: $ADDRESS_CITY_STATE_ZIP
           required: true
+        - name: Phone
+          selector: div .phone input
+          value: $PHONE
+          required: false
+        - name: Subject
+          selector: div .messagesubject input
+          value: $SUBJECT
+          required: false
         - name: message
           selector: div.dearsenatorvinehout textarea
           value: $MESSAGE
           required: true
     - click_on:
-        - selector: div#contour_form_Contact input[name='submitbtn']
+        - selector: input[type='submit'][value='Submit']
+    - wait:
+        - value: 3
     - find:
-        - selector: p.contourMessageOnSubmit
+        - selector: div .panel-heading
   success:
     headers:
       status: 200
     body:
-      contains: "Thank you"
+      contains: "Feature News"
```

Comment:
same as above re: unique message

Reply:
This one is kinda tricky. There's no success message and after submitting the form the user is redirected to the home page (https://gyazo.com/bc4b8093bff7edcc47c80752a02dcc21). Of all the text in this page, I thought 'Feature News' was the only constant one that would't change. I also thought about using "Madison Office" (left hand corner), but that one seems to be present in every page you navigate to.  Anything else you see there that I could use?

Reply:
ohh man the photo of that kid on the home page.... the most awkward and also the best.

as long as `feature news` will never be present on the page where the form is, that's fine then.

Reply:
yeah poor kid lol and I didn't notice the hemp agriculture piece, that's really interesting lol